core/src/droplet/libdroplet/src/backend/cdmi/replyparser.c

```diff
@@ -586,7 +586,7 @@ dpl_status_t dpl_cdmi_parse_list_bucket(dpl_ctx_t* ctx,
     goto end;
   }
 
-  children = json_object_object_get(obj, "children");
+  json_object_object_get_ex(obj, "children", &children);
```

Comment:
return code is book and should probably be checked. The docs doesn't state what `children` will be set to on error, so we should probably assume it can be undefined and not rely on comparing with `NULL` only.